components/weather_data/WeatherStationList.tsx

```diff
@@ -54,7 +139,193 @@ const StationList: React.FunctionComponent<{
   if (center_id === 'NWAC') {
     return <NWACStationList center={center_id} mapLayer={mapLayer} stations={weatherStations} requestedTime={requestedTime} />;
   }
-  return <Unavailable />;
+
```

Comment:
Can this code get moved into a separate component, so that we have something more like `return <WeatherStationMap {...whateverPropsItNeeds} />`? 

This component has basically been responsible for fetching the data and instantiating the correct component to display it, or displaying a `QueryState` if it's not ready yet - that seems like a good separation of concerns. Now it's learning about the curvature of the earth which is quite a leap :)